apps/web/pages/[user].tsx

```diff
@@ -92,7 +92,13 @@ export function UserPage(props: InferGetServerSidePropsType<typeof getServerSide
             "max-w-3xl px-4 py-24"
           )}>
           <div className="mb-8 text-center">
-            <Avatar imageSrc={profile.image} size="xl" alt={profile.name} />
+            <Avatar
+              imageSrc={profile.image}
+              size="xl"
+              alt={profile.name}
+              organizationLogo={profile.organizationLogo}
+              organizationName={profile.organizationName}
+            />
```

Comment:
How about creating an HOC <BigAvatar> that can make `organizationLogo` and `organizationName` required props. Right now these are optional props because other sizes avatars don't need org logo.

We can pass explicit null when user isn't org member. This would help in ensuring that we never miss org logo with big avatars.

Reply:
I envision this as 
![image](https://github.com/calcom/cal.com/assets/55134778/41be4db3-ac3d-423d-bc22-3656fd551731)
Very common for avatars to have a so called indicator e.g we use it for online/away status in user dropdown.

Or something like how chakra-ui does this
![CleanShot 2023-08-11 at 15 35 56](https://github.com/calcom/cal.com/assets/55134778/ead5a46b-2e9c-455d-aa03-2959aa3a52bf)

Reply:
I was going to suggest the same thing as Sean, we should probably let Avatar component put any kind of bottom badge so we can cover online status and org logo. I think making it configurable from the outside should be enough, so you can pass the avatar either the org logo or the online status as an additional property and if that's present, put it as is in the badge area.

Reply:
Stacked PR addressing this feedback
https://github.com/calcom/cal.com/pull/10788